tests/testsuite/build.rs

```diff
@@ -2154,6 +2154,57 @@ fn crate_library_path_env_var() {
     setenv_for_removing_empty_component(p.cargo("run")).run();
 }
 
+// See https://github.com/rust-lang/cargo/issues/14194
+#[cargo_test]
+fn issue_14194_deduplicate_library_path_env_var() {
+    let p = project()
+        .file(
+            "src/main.rs",
+            &format!(
+                r#"
+                    use std::process::Command;
+                    fn main() {{
+                        let args: Vec<String> = std::env::args().collect();
+                        if args.len() == 1 {{
+                            return;
+                        }}
+                        let level: i32  = args[1].parse().unwrap();
```

Comment:
This is probably better?

```suggestion
                        let level: i32 = std::env::args().nth(1).unwrap().parse().unwrap();
```